### ISSUE TITLE
`movementSpeed` attribute (and all attributes other than `attackSpeed` and `attackDamage`) not showing up in lore

### ISSUE THREAD

**Issue by yorik100** [PluginBugs/Issues-ItemsAdder#2711]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### Discord tag (optional)

yorik100#5011

### What happened?

If you have a weapon with attackdamage, attackspeed and speed, speed wont be shown in lore

### Steps to reproduce the issue

1. Create a weapon with attakdamage, attackspeed and speed
2. Hover your cursor over that weapon
3. Speed attribute doesn't show

### Server version

Paper version git-Paper-546 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT)

### ItemsAdder Version

3.4.1f2

### ProtocolLib Version

5.1.0

### LoneLibs Version

1.0.24

### FULL server log

Not needed

### Screenshots/Videos (you can drag and drop files or paste links)

In game :
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/16320315/3cb75f10-5b1e-45dc-89bb-a38710a041d2)
In settings :
![image](https://github.com/PluginBugs/Issues-ItemsAdder/assets/16320315/856aabc0-62cb-421b-b5ff-af1bf7d29f7e)

Speed boost is getting applied, it's just not showing up in lore

**Comment by LoneDev6** [comment]
Yes this is normal, for now you have to set it in the lore manually.
I'll try to see if I can code a fix for that but could be tricky.

Thanks

**Comment by yorik100** [comment]
> Yes this is normal, for now you have to set it in the lore manually. I'll try to see if I can code a fix for that but could be tricky.
> 
> Thanks

Remember that other attributes exist, including knockback resistance and bonus health

**Comment by LoneDev6** [comment]
More info here: https://github.com/PluginBugs/Issues-ItemsAdder/issues/448#issuecomment-792871597

> You have to disable this in config.yml of ItemsAdder.
> `fix-show-attack-and-damage-as-lore: false`
> 
> ItemsAdder has to do some tricks to show the correct value for damage and speed of weapons because the game uses a stupid calculation and would show slightly imprecise values (+4 for speed and +1 for damage).
> 
> To do that ItemsAdder hides the vanilla information using the item flag `HIDE_ATTRIBUTES`, but the downside is that it hides all attributes not only speed and damage.
> So I have to manually write these in the lore, but I prefer not to write ALL possible attributes for performance reasons.
> If you have more than speed and damage attributes you can write them on your own using the lore attribute of the ItemsAdder item.
> 
> Let me know if the solution is good or not.
